src/ai/backend/manager/cli/__main__.py

```diff
@@ -109,6 +113,64 @@ def generate_keypair(cli_ctx: CLIContext):
     print(f'Secret Key: {sk} ({len(sk)} bytes)')
 
 
+@main.command()
+@click.option('-r', '--retention', type=str, default='1yr',
+              help='The retention limit. e.g., 20d, 1mo, 6mo, 1yr')
+@click.option('-v', '--vacuum-full', type=bool, default=False,
+              help='Reclaim storage occupied by dead tuples.'
+                    'If not set or set False, it will run VACUUM without FULL.'
+                    'If set True, it will run VACUUM FULL.'
+                    'When VACUUM FULL is being processed, the database is locked.'
+                    '[default: False]')
+@click.pass_obj
+def clear_history(cli_ctx: CLIContext, retention, vacuum_full) -> None:
+    """
+    Delete old records from the kernels table and
+    invoke the PostgreSQL's vaccuum operation to clear up the actual disk space.
+    """
+    local_config = cli_ctx.local_config
+    with cli_ctx.logger:
+        unit = retention[-2:]
+        today = datetime.now()
+        if unit == 'yr':
+            yr = int(retention[:-2])
+            expiration_date = today - relativedelta(years=yr)
+        elif unit == 'mo':
+            mo = int(retention[:-2])
+            expiration_date = today - relativedelta(months=mo)
+        else:
+            duration = TimeDuration()
+            expiration_date = today - duration.check_and_return(retention)
+
+        conn = psycopg2.connect(host=local_config['db']['addr'][0],
+                                port=local_config['db']['addr'][1],
+                                dbname=local_config['db']['name'],
+                                user=local_config['db']['user'],
+                                password=local_config['db']['password'])
+        with conn.cursor() as curs:
+            if vacuum_full:
+                vacuum_sql = "VACUUM FULL"
+            else:
+                vacuum_sql = "VACUUM"
+
+            log.info('Deleting old records...')
+            curs.execute(f"""
+            DELETE FROM kernels WHERE terminated_at < '{expiration_date.strftime('%Y-%m-%d %H:%M:%S')}';
+            """)
+            log.info(f'Perfoming {vacuum_sql} operation...')
+            conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
+            curs.execute(vacuum_sql)
+            conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_READ_COMMITTED)
+
+            curs.execute("""
+            SELECT COUNT(*) FROM kernels;
+            """)
+            table_size = curs.fetchone()
+            log.info(f'kernels table size: {table_size[0]}')
+
+        log.info('Clear up operation is done.')
```

Comment:
Let's explicitly print out the calculated expiration date and the number of removed items by counting them before deletion, e.g.,
```suggestion
        log.info('Cleaned up {:,} database records older than {:%Y-%m-%d %H:%M:%S}.', deleted_count, expiration_date)
```